Cargo.toml

```diff
@@ -11,3 +11,29 @@ members = [
     "mgm",
 ]
 resolver = "2"
+
+[patch.crates-io]
+
+# https://github.com/RustCrypto/block-ciphers/pull/413
+aes            = { git = "https://github.com/RustCrypto/block-ciphers.git" }
+
+# https://github.com/RustCrypto/block-modes/pull/56
+ctr            = { git = "https://github.com/RustCrypto/block-modes.git" }
+
+# pending pre-release
+universal-hash = { git = "https://github.com/RustCrypto/traits.git" }
+# https://github.com/RustCrypto/traits/pull/1531
+aead           = { git = "https://github.com/RustCrypto/traits.git" }
+
+# https://github.com/RustCrypto/MACs/pull/158
+cmac           = { git = "https://github.com/RustCrypto/MACs.git" }
+pmac           = { git = "https://github.com/RustCrypto/MACs.git" }
+
+# https://github.com/RustCrypto/stream-ciphers/pull/345
+chacha20       = { git = "https://github.com/RustCrypto/stream-ciphers.git" }
+
+# https://github.com/RustCrypto/universal-hashes/pull/196
+ghash          = { git = "https://github.com/RustCrypto/universal-hashes.git" }
+polyval        = { git = "https://github.com/RustCrypto/universal-hashes.git" }
+poly1305       = { git = "https://github.com/RustCrypto/universal-hashes.git" }
```

Comment:
Perhaps we could kick out some releases of these so we have a patch-free baseline to work from

Reply:
that'd be lovely!

In order, I think I'd need in order:
 - First step:
   - traits/aead
   - traits/universal-hash
   - block-ciphers/aes
   - block-ciphers/belt-block
   - block-ciphers/des
   - block-ciphers/kuznyechik
   - block-ciphers/magma
   - stream-ciphers/chacha20
 - second 
   - macs/cmac
   - macs/pmac
   - block-modes/ctr
   - universal-hashes/polyval
   - universal-hashes/poly1305
 - third
   - universal-hashes/ghash

Reply:
that's a lot of releases! I can throw in the first steps of those in a couple PRs if you'd like

Reply:
Sure